### PR TITLE
Standardize redux store + Reorganize dashboard jsx

### PR DESCRIPTION
**Pull request by nareddyt**

### CODE REVIEW

File: src/redux/actions.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export function updateUser (user) {
```
Comment (ganhari123) [inline src/redux/actions.js]: I would make all the actions constants for good practice and access those constants in reducer

File: src/dash/student_view/Profile.jsx [hunk 2]
```diff
@@ -1,48 +1,28 @@
 import React from 'react';
 import { connect } from 'react-redux';
```
Comment (ganhari123) [inline src/dash/student_view/Profile.jsx]: Be careful editting anything on profile why @SammyBaek is working on it as well

Comment (nareddyt) [reply src/dash/student_view/Profile.jsx]: yup!

Comment (nareddyt) [reply src/dash/student_view/Profile.jsx]: I believe he is working off this branch now so we should be good. I took all his changes this morning so there should be no conflicts if he didn't work on it today

File: src/redux/actions.js (hunk 1 above)
Comment (nareddyt) [reply src/redux/actions.js]: How do we do that?

**Pull request merged**